### PR TITLE
chore : refacto of i18n coding

### PR DESCRIPTION
**Pull request by FlorentMr**
## Describe your changes
chore : refacto of i18n coding

## Checklist tests
Check if i18n (ours and Edifice ones in ShareModal for example) are working, in at least FR and EN, in the application and in the timeline.

## Issue ticket number and link
[MAG-436](https://jira.support-ent.fr/browse/MAG-436)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/tree-view/TreeViewContainer.tsx [hunk 1]
```diff
@@ -37,324 +37,326 @@ export const TreeViewContainer: React.FunctionComponent<
...
+          onDragOver={handleDragOver}
+          onDragEnter={handleDragEnter}
+          onDragLeave={handleDragLeave}
+          onDrop={handleDrop}
+        >
+          <TreeView
+            selectedNodesIds={selectedNodeIds}
+            data={datas || dataTree}
+            onTreeItemBlur={() => {
+              console.log("blur");
```
Comment (niko-amosse) [inline frontend/src/components/tree-view/TreeViewContainer.tsx:344]: console.log à enlever

File: frontend/src/components/tree-view/TreeViewContainer.tsx [hunk 2]
```diff
@@ -37,324 +37,326 @@ export const TreeViewContainer: React.FunctionComponent<
...
+            }}
+            onTreeItemFocus={() => {
+              console.log("focus");
```
Comment (niko-amosse) [inline frontend/src/components/tree-view/TreeViewContainer.tsx:347]: idem

**Review by niko-amosse — APPROVED** [review]

File: backend/src/main/resources/i18n/fr.json [hunk 3]
```diff
@@ -0,0 +1,210 @@
+{
```
Comment (alicedraillard) [inline backend/src/main/resources/i18n/fr.json:1]: pourquoi ce fichier a changé alors que les autres non ?

File: frontend/src/i18n.ts [hunk 4]
```diff
@@ -1,30 +1,38 @@
-import i18n, { Resource } from "i18next";
+import i18n from "i18next";
 import LanguageDetector from "i18next-browser-languagedetector";
+import Backend from "i18next-http-backend";
 import { initReactI18next } from "react-i18next";
 
-import { dictionaries } from "~/dictionaries";
```
Comment (alicedraillard) [inline frontend/src/i18n.ts:5]: Rajouter une partie "i18n Edifice" dans la doc en précisant pk on en a besoin
https://confluence.support-ent.fr/display/DN/Mettre+en+place+les+i18n+dans+un+projet

File: backend/src/main/resources/i18n/fr.json (hunk 3 above)
Comment (FlorentMr) [reply backend/src/main/resources/i18n/fr.json:1]: Je crois que c'est parce que je l'ai recréé à la main et pas déplacé, mais c'est la même chose que les autres no soucis

**Pull request merged**